### PR TITLE
Update custom-skins.md

### PR DESCRIPTION
**Pull request by pronoymukherjeewritescode**

### CODE REVIEW

File: docs/articles/custom-skins.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Skins are loaded from the ```Skins``` folder in the app's ```LocalState``` folde
 
 ### Quick start
 
-1. Open your text editor of choice, and paste in the template located below. Then, save the file as ```%userprofile%\AppData\Local\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState\Skins\test1.xaml```.
+1. Open your text editor of choice, and paste in the template located below. Then, save the file as ```%userprofile%\AppData\Local\Packages\Files_1y0xx7n9077q4\LocalState\Themes```.
```
Comment (chingucoding) [inline docs/articles/custom-skins.md:9]: This path doesn't work on my machine. @yaichenbaum Should the new proposed path work?

Comment (yaichenbaum) [reply docs/articles/custom-skins.md:9]: I'm guessing @pronoymukherjeewritescode is running our sideload version for v1.5 which has a different path. In any case, the current doc is the correct path for the Store version and should be kept how it is for now.

Comment (chingucoding) [reply docs/articles/custom-skins.md:9]: What would you advise to do now @yaichenbaum ? Leave docs as is?

Comment (yaichenbaum) [reply docs/articles/custom-skins.md:9]: Yeah.

**Comment by chingucoding** [comment]
Thank you for your contribution @pronoymukherjeewritescode, however right now, the documentation around this is fine as is. Because of this, I will close this PR now.

**Pull request closed without merging**